{"name": "Non auth user data update failed test", "status": "passed", "start": 1732664554289, "stop": 1732664555067, "uuid": "d481ceb0-676c-4601-80ef-97961db38126", "historyId": "207907099f99c9c8590beda7b51e8b3e", "testCaseId": "207907099f99c9c8590beda7b51e8b3e", "fullName": "tests.test_update_user_data.TestUpdateUserData#test_update_user_data_non_auth_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_update_user_data"}, {"name": "subSuite", "value": "TestUpdateUserData"}, {"name": "host", "value": "DESKTOP-V6N82FL"}, {"name": "thread", "value": "24496-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_update_user_data"}]}